hs_core/hydroshare/utils.py

```diff
@@ -856,7 +856,7 @@ def get_party_data_from_user(user):
 
     party_data['name'] = party_name
     party_data['email'] = user.email
-    party_data['description'] = '/user/{uid}/'.format(uid=user.pk)
+    party_data['hydroshare_user_id'] = user.pk
```

Comment:
Looks like you got just about everything.  The last thing to consider is the bag and how this might affect hsmodels.  We can jump on a huddle this week to go over that.

Reply:
thanks @sblack-usu 
PRs:
https://github.com/hydroshare/hsmodels/pull/32
https://github.com/hydroshare/hsclient/pull/40
I guess I still feel like this change is going to cause headaches for some people.
For example, what if someone downloaded a Bag today and then tried to ingest it into HS after 1.55 release?
They wouldn't be able to because their bagit xmls wouldn't line up with our new Party class (due to replaced "description").
Should we consider keeping the 'description' for backward compatibility or are we not concerned? Thanks